.github/workflows/ci.yml

```diff
@@ -76,8 +71,48 @@ jobs:
       - name: Run integration tests to the compiler itself
         run: vendor/bin/phpunit --testsuite integration
 
+  core-tests:
+    name: Core tests on PHP ${{ matrix.php }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        php: [ '7.4', '8.0' ]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          coverage: none
+
+      - name: Install dependencies
```

Comment:
Is it possible to install dependencies only once? I'm more aware with GitLab Pipelines than with GitHub. In GitLab you can cache the vendor dir and use it in other task again.

Reply:
It might/should be possible, but I don't know it right now. I will investigate that and come back to this.